src/Config.js

```diff
@@ -72,6 +73,7 @@ export class Config {
     readOnlyMasterKey,
     allowHeaders,
     idempotencyOptions,
+    graphQLConfig,
```

Comment:
Could you rename the parameter to `GraphqlConfig` and the internal var to `graphqlConfig`? We are generally using camel case, so if we consider GraphQL to be a "brand name", whether it's written "glraPhqL" or however, we would only capitalize the first letter so that people don't have to lookup each option how it's written. A similar legacy example is `publicServerURL` which should be `publicServerUrl`.